CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
 # Contributing
+> Please note that this project is released with a [Contributor Code of Conduct](#CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
 Thanks for your interest in contributing to this project. You can contribute or report issues in the following ways:
```

Comment:
Could you add a newline between both of these? Currently the intro text is being combined with the notice when [viewing on GitHub](https://github.com/Stormy102/CSSS/blob/feature/ReadMeBadgify/CONTRIBUTING.md#contributing).